src/cdk-experimental/menu/menu-item.ts

```diff
@@ -50,9 +53,51 @@ export class CdkMenuItem {
   }
   private _checked = false;
 
+  /**  Whether the CdkMenuItem is disabled - defaults to false */
+  @Input()
+  get disabled(): boolean {
+    return this._disabled;
+  }
+  set disabled(value: boolean) {
+    this._disabled = coerceBooleanProperty(value);
+  }
+  private _disabled = false;
+
   /** Emits when the attached submenu is opened */
   @Output() opened: EventEmitter<void> = new EventEmitter();
 
+  constructor(
+    /** reference a parent CdkMenuGroup component */
+    private readonly _menuGroup: CdkMenuGroup
```

Comment:
I know we discussed this at some point, but we can do it more here- where did we land on making it so that the menu/items only know about each other in one direction? E.g. the menu know about its items OR the items know about their menu, but not both.

Reply:
We decided to use an interface which I've started implementing for the next feature in order to prevent circular dependencies